messages/fr.json

```
@@ -0,0 +1,27 @@
+{
+    "Home": {
+        "enter": "Entrer"
+    },
+    "LocaleSwitcher": {
+        "en": "Anglais",
+        "es": "Espagnol",
+        "fr": "Français",
+        "label": "Langue"
```

Similar thing here, not 100% sure but my instinct is that this should probably always be in English if it's what's defining the selector.

```suggestion
        "label": "Language"
```